springwolf-plugins/springwolf-kafka-plugin/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/operationdata/annotation/KafkaAsyncOperationBinding.java

```diff
@@ -20,4 +20,21 @@
     String clientId() default "";
     String bindingVersion() default "";
 
+    KafkaAsyncMessageBinding messageBinding() default @KafkaAsyncMessageBinding();
+
+    @Retention(RetentionPolicy.CLASS)
+    @Target({})
+    @interface KafkaAsyncMessageBinding {
+
+        KafkaKeyTypes keyType() default KafkaKeyTypes.NO_KEY;
+
+        String description() default "";
```

Comment:
Description? https://github.com/asyncapi/bindings/blob/master/kafka/README.md#message

Reply:
Should there be a sub object for the key only, which combines type and description?

Reply:
> Description? https://github.com/asyncapi/bindings/blob/master/kafka/README.md#message
Yes, I think it's valid because key is actually a schema and therefore may contain more data: https://www.asyncapi.com/docs/reference/specification/v2.4.0#schemaObject 
I was also thinking about using a schema reference, but I'm not sure how much use there would be and it could potentially pollute the "schema view" ...